cluster/examples/kubernetes/ceph/psp.yaml

```diff
@@ -0,0 +1,133 @@
+---
+
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: rook-privileged
+spec:
+  # TODO: can fsGroup be trimmed down?
```

Comment:
Shall we remove the TODOs? They aren't going to be resolved anytime soon.

Reply:
Being in a config file, I was hoping a user with knowledge of how to trim the permissions down might see that and offer some help. I could remove `TODO` but still leave a note that the permissions aren't fully trimmed down?

Reply:
What about a brief comment about why they are currently necessary? Then the motivated contributor might try to address trimming them down.